docs/installation.md

```diff
@@ -0,0 +1,69 @@
+# Installation
+
+There are two basic setups that can be put in place in order to develop and test Paricia locally:
+
+- Virtual environment, used for daily development of the code and the documentation. See section on the [virtual environment setup](#virtual-environment).
+- Docker, used to run the tool locally, accessible in the browser as well as to run tests. See section on [docker](#docker-deployment).
+
+## Virtual environment
+
+The normal software development should be done within a virtual environment, where the tools and all the dependencies that Paricia requires are installed. This enables to run the appropriate linters, code formatters, and autocompletion features of the code editor specifically for Paricia. Additionally, it also let you create and develop the documentation.
+
+To setup a virtual environment with all the requirements, navigate to Paricia's root directory in a terminal and run (this should work in all platforms):
+
+```bash
+python -m venv .venv
+```
+
+This will create an isolated Python environment within a directory called `.venv`. Notice that Paricia requires Python 3.11 or higher to work. Once the environment has been created, you can ativate it with:
```

Comment:
```suggestion
This will create an isolated Python environment within a directory called `.venv`. Notice that Paricia requires Python 3.11 or higher to work. Once the environment has been created, you can activate it with:
```